CHANGELOG.md

```diff
@@ -72,9 +72,13 @@ To release a new version, please update the changelog as followed:
 - Repository:
   - Danger CI has been added to enforce the update of the changelog (by @lgarithm and @DEKHTIARJonathan)
 - Tutorials:
-  - `tutorial_tfslim` added by (@2wins).
+  - `tutorial_tfslim` added (by @2wins).
+- Tests:
+  - `test_utils_predict.py` added (by @2wins).
```

Comment:
Please mention why the test has been added: to insure that issue #??? is solved.

Reply:
You can do the following:

```
- `test_utils_predict.py` added to reproduce and fix issue #288 (by @2wins).
```